deploy/env/e2e-tests.yaml

```
@@ -2,13 +2,10 @@ registration-service:
   environment: 'e2e-tests'
   replicas: 2
   verification:
-    #enabled: 'true'
-    enabled: 'false'
+    enabled: 'true'
     excluded-email-domains: 'redhat.com,acme.com'
```

I wonder if we use something more obvious for the excluded email domains in our tests. Something like:
```suggestion
    excluded-email-domains: 'redhat-exclude-from-verification.com,exclude.com'
```
WDYT?